internal/pally/registry.go

```diff
@@ -70,14 +69,14 @@ func Federated(prom prometheus.Registerer) RegistryOption {
 }
 
 // Labeled adds constant labels to a Registry. All metrics created by a
-// Registry inherit its constant labels.
+// Registry inherit its constant labels. Labels with invalid names or values
+// are dropped.
 func Labeled(ls Labels) RegistryOption {
+	// TODO: Consider whether we should automatically scrub
+	// statically-specified label values.
```

Comment:
+1 for automatically scrubbing, labels simply disappearing will cause a lot more head scratching.